### PR TITLE
Fix: optimize  backgrounds

### PR DESCRIPTION
**Pull request by sorodrigo**
This PR optimizes some of the images used across the app and adds the ImgBackground component to entrypoints.

### CODE REVIEW

File: frontend/scripts/react-components/shared/img/img.component.jsx [hunk 1]
```diff
@@ -29,7 +29,15 @@ function Img(props) {
         return (
           <>
             <img ref={ref} alt={alt} sizes={sizes} srcSet={srcSetProp} src={srcProp} {...rest} />
-            {!inView && <a href={src}>{alt}</a>}
+            {!inView && (
```
Comment (Bluesmile82) [inline frontend/scripts/react-components/shared/img/img.component.jsx]: Shouldn't we keep the alt for SEO?

File: frontend/public/images/dashboards/icon_commodities.svg [hunk 2]
```diff
@@ -1,23 +1 @@
-<?xml version="1.0" encoding="UTF-8"?>
```
Comment (Bluesmile82) [inline frontend/public/images/dashboards/icon_commodities.svg]: We can actually use https://github.com/tcoopman/image-webpack-loader to optimize SVGs with SVGO

File: frontend/scripts/react-components/shared/img/img.component.jsx (hunk 1 above)
Comment (sorodrigo) [reply frontend/scripts/react-components/shared/img/img.component.jsx]: it's still there in line 38 😛

Comment (Bluesmile82) [reply frontend/scripts/react-components/shared/img/img.component.jsx]: Ok, monday reading 🤦‍♂ Good one!

File: frontend/public/images/dashboards/icon_commodities.svg (hunk 2 above)
Comment (sorodrigo) [reply frontend/public/images/dashboards/icon_commodities.svg]: Nice! Though we'd need to test it works without importing the images like
```js
import myImage from '../images'
```

Comment (sorodrigo) [reply frontend/public/images/dashboards/icon_commodities.svg]: We already have it on prod, but not sure it's working and it's probably due to how we load the svgs. Let's review that in the future.

**Pull request merged**